docs/05-technical-reference/00-custom-resources/svls-01-function.md

```diff
@@ -100,7 +100,7 @@ This table lists all the possible parameters of a given resource together with t
 | ---------------------------------------- | :------------: | ---------|
 | **metadata.name**              |      Yes       | Specifies the name of the CR.                 |
 | **metadata.namespace**     |       No       | Defines the Namespace in which the CR is available. It is set to `default` unless you specify otherwise.      |
-| **spec.env**                             |       No       | Specifies environment variables you need to export for the Function. You can export them either directly in the Function CR's spec or define them in a [ConfigMap](#configuration-environment-variables-define-environment-variables-in-a-config-map). |
+| **spec.env**                             |       No       | Specifies environment variables you need to export for the Function. You can export them either directly in the Function CR's spec or define them in a ConfigMap. |
```

Comment:
The doc that it linked to is now in **Technical References** > **Configuration Parameters** > `svls-02-environment-variables.md` > section **Define  environment variables in a ConfigMap**. 
Here's the link:
```suggestion
| **spec.env**                             |       No       | Specifies environment variables you need to export for the Function. You can export them either directly in the Function CR's spec or define them in a [ConfigMap](../../00-configuration-parameters/svls-02-environment-variables.md#define-environment-variables-in-a-configmap). |
```

Reply:
Thank you, I totally missed it! 🙇